client/disputes/evidence/index.js

```diff
@@ -443,6 +443,7 @@ export default ( { query } ) => {
 				? 'wcpay_dispute_submit_evidence_success'
 				: 'wcpay_dispute_save_evidence_success'
 		);
+		// eslint-disable-next-line inclusive-language/use-inclusive-words
```

Comment:
The `woocommerce-admin` repo will move its default branch to `trunk` soon (same with Gutenberg probably), so we'll be able to change the links and remove this silliness.

Reply:
They did already, I'll update URLs accordingly.